packages/trader/src/Modules/Reports/Components/account-statistics.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { MobileWrapper, Money } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const AccountStatistics = (account_statistics, currency) => (
```

Comment:
Since this is a component, you should breakdown your props here instead of parameters